erpnext/patches/v13_0/replace_supplier_item_group_with_party_specific_items.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2019, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+
+import frappe
+
+def execute():
+	sig = frappe.db.get_all("Supplier Item Group", fields=["name", "supplier", "item_group"])
```

Comment:
This doctype was recently added so patches will fail unless you do `reload_doc` on all the relevant doctypes.